plugin/src/main/java/org/openrewrite/gradle/RewriteDryRunTask.java

```diff
@@ -34,7 +32,15 @@ public class RewriteDryRunTask extends AbstractRewriteTask {
     // On Gradle 4.0 annotating something returning a Path with @OutputFile triggers a bug that deadlocks Gradle
```

Comment:
Does the above comment apply to the whole 4.x line? Or can this be revised now too?

Reply:
Good question. When I change it to return `Path` and run `./gradlew check`, it fails with an "unexpected build error". Let me see if I can debug this further...

Reply:
Okay, I'm not sure what happened. It might've been a symptom, as my machine has so many CPU cores that rewrite-gradle-plugin's test suite spins up enough JVMs to exhaust my RAM sometimes.

Anyway, migrating `File` to `Path` apparently works on Gradle 4.10, so I'm about to push a change soon which will invalidate this comment. Thanks again for catching it!